app/(main)/markets/News.tsx

```diff
@@ -77,6 +80,7 @@ const NewsArticle = ({ url, title }: NewsArticleProps) => {
       href={url}
       target="_blank"
       rel="noopener noreferrer"
+      onClick={() => trackEvent(FA_EVENTS.MARKETS.DETAILS.TABS.NEWS_ARTICLE)}
```

Comment:
we can pass an article title tho

Reply:
Yes! We can. I'm not sure if it's worth it to track which title the user clicks, as each market will have its own articles, so there will be a lot of single click events in market news that are closed already